src/python/gem5/components/boards/lupv_board.py

```diff
@@ -143,9 +158,16 @@ def __init__(
             self._int_ids['BLK'],
             self._int_ids['RNG']
         ]
-        self.pic.n_contexts = self.processor.get_num_cores() * 2
-        self.pic.n_src = max(pic_srcs) + 1
 
+        # Set the number of sources to the pic as 0 since we only have the
+        # PIC and CLINT connected to the board for our bbl to use upon 
+        # startup, and they will remain unused during the simulation
```

Comment:
Well, technically the number of sources is 0 because there are no external devices actually connected to the PIC.